app/commands/rename_data_category_command.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class RenameDataCategoryCommand < ApplicationCommand
+  usage do
+    no_command
+  end
+
+  keyword :id do
+    desc "The target category ID"
+    required
+    validate { _1 =~ Category::ID_REGEX }
+  end
+
+  keyword :name do
+    desc "Updated category name"
+    required
+  end
+
+  def execute
+    frame("Adding new category") do
+      find_category!
+      update_category!
+      update_vertical_file!
+      sync_localizations!
+    end
+  end
+
+  private
+
+  def find_category!
+    @category = Category.find_by(id: params[:id])
+    @original_handle = @category&.handleized_name
+    return if @category
+
+    logger.fatal("Category `#{params[:id]}` not found")
+    exit(1)
+  end
+
+  def update_category!
+    spinner("Updating category") do |sp|
+      original_name = @category.name
+      @category.update!(name: params[:name])
+
+      sp.update_title("Updated category `#{original_name}` to `#{params[:name]}`")
+    end
+  end
+
+  def update_vertical_file!
+    spinner("Updating vertical file") do |sp|
+      if @category.root?
+        sys.move_file!("data/categories/#{@original_handle}.yml", "data/categories/#{@category.handleized_name}.yml")
```

Comment:
I think this should be logged when it happens so the user is informed that the file is being renamed